impeller/typographer/backends/skia/text_frame_skia.h

```diff
@@ -10,6 +10,7 @@
 
 namespace impeller {
 
-TextFrame MakeTextFrameFromTextBlobSkia(const sk_sp<SkTextBlob>& blob);
+std::optional<TextFrame> MakeTextFrameFromTextBlobSkia(
```

Comment:
I made this optional so that we can avoid creating TextContents at all if the textblob is nullptr for some reason.

Reply:
Ditto above, comment in the code :)